beast/observationmodel/observations.py

```diff
@@ -248,6 +250,14 @@ def gen_SimObs_from_sedgrid(
         Filter to use for completeness (required for toothpick model).
         Set to max to use the max value in all filters.
 
+    complcut : float (defualt=None)
+        completeness cut for only including model seds above the cut
+        raning from 0.0 to 1.0.
```

Comment:
raning -> ranging
Maybe "where the completeness ranges between 0 and 1."

Reply:
Fixed.